src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/OperatorWrapCheck.java

```diff
@@ -94,6 +94,37 @@
  * &lt;module name="OperatorWrap"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * class Test {
+ *     public static void main(String[] args) {
+ *         String s = "Hello" +
+ *         "World"; // violation, '+' should be on new line
```

Comment:
`// violation, '+' should be on new line` violation is reported on '+' token. Violation comment should be on same line as violation reported

Reply:
I cannot put th comment  on the above line because there will be a comment node ahead which will not violate the rule @strkkk

Reply:
I do not understand. I mean to do it like
``` String s = "Hello" + // violation, '+' should be on new line
         "World"; ```

Reply:
Yes if i do that there no violation i tries this @strkkk

Reply:
it looks like a bug to me.
@romani @rnveach @pbludov 
Is OperatorWrapCheck should work like this? I mean, do not report violation in case like 
```
String s = "Hello" + // should be violation, but not
         "World"; 
```
seems weird.
Check only checks if rest of line after operator is blank, but single comments should be ignored as well.

Reply:
@harsh-kukreja anyway, then please ignore items about moving this comment.

Reply:
Should i create a Pull request regarding this?

Reply:
@harsh-kukreja yes, please. It is a good catch.